### PR TITLE
feat: implement Tile component on @dashboard module

### PR DESCRIPTION
**Pull request by yvmunayev**
fix: #100 

## Changes proposed in this PR:
- feat: implement Tile component on @dashboard module

### CODE REVIEW

File: packages/dashboard/src/components/Tile/checkmark.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Checkmark(props) {
+    const { className, style } = props;
```
Comment (TahimiLeonBravo) [inline packages/dashboard/src/components/Tile/checkmark.js:5]: I think we should consider using the ones on icons module

**Review by TahimiLeonBravo — CHANGES_REQUESTED** [review]

File: packages/dashboard/src/components/Tile/index.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+    const type = multiple ? 'checkbox' : 'radio';
+    const name = groupName || nameProp;
+    const checked = isChecked(multiple, value, nameProp);
+
+    const labelStyle = { color: replaceAlpha(colorToRgba(color), 0.7) };
+
+    return (
+        <StyledContainer id={id} className={className} style={style}>
+            <StyledInput
```
Comment (TahimiLeonBravo) [inline packages/dashboard/src/components/Tile/index.js:40]: I think we should not use an input when the component is presentational, it only should be used as an input with `focus` and `selected` styles when it is used inside the `TilePicker` component

Comment (TahimiLeonBravo) [reply packages/dashboard/src/components/Tile/index.js:40]: @maxxgreene @LeandroTorresSicilia I think we should discuss that

**Review by TahimiLeonBravo — CHANGES_REQUESTED** [review]

File: packages/dashboard/src/components/Tile/index.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+                    as="input"
+                    type={type}
+                    id={inputId}
+                    name={name}
+                    checked={checked}
+                    aria-describedby={ariaDescribedby}
+                    onChange={(event) => onChange(nameProp, event.target.checked)}
+                    value={valueProp}
+                />
+                <StyledLabelInput htmlFor={inputId} style={{ backgroundColor }}>
```
Comment (TahimiLeonBravo) [inline packages/dashboard/src/components/Tile/index.js:53]: let's use `StyledLabel` instead `StyledLabelInput`

File: packages/dashboard/src/components/Tile/index.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+                    <RenderIf isTrue={checked}>
+                        <StyledCheckmarkContainer>
+                            <Check style={{ color }} />
+                        </StyledCheckmarkContainer>
+                    </RenderIf>
+                    <StyledLabel style={labelStyle}>{label}</StyledLabel>
```
Comment (TahimiLeonBravo) [inline packages/dashboard/src/components/Tile/index.js:59]: let's use `StyledLabelText` instead `StyledLabel`

**Review by TahimiLeonBravo — CHANGES_REQUESTED** [review]

**Comment by TahimiLeonBravo** [comment]
@yvmunayev add documentation for the component

File: packages/dashboard/src/components/Tile/helpers/__test__/isChecked.spec.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import isChecked from '../isChecked';
+
+describe('isChecked', () => {
+    it('should return true when is multiple and name is include in value', () => {
+        expect(isChecked(true, [0, 1, 2], 1)).toBe(true);
```
Comment (LeandroTorresSicilia) [inline packages/dashboard/src/components/Tile/helpers/__test__/isChecked.spec.js:5]: in order to improve readability, we should use only one argument as an object thus we can add names to the parameters like:
```icChecked({
    multiple: true,
    value: [...],
    name: 1,
    ...
})```

File: packages/dashboard/src/components/Tile/helpers/__test__/isChecked.spec.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    });
+
+    it('should return false when is multiple and name is include in value', () => {
```
Comment (LeandroTorresSicilia) [inline packages/dashboard/src/components/Tile/helpers/__test__/isChecked.spec.js:8]: not include

File: packages/dashboard/src/components/Tile/index.js [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+            </RenderIf>
+        </StyledContainer>
+    );
+}
+
+Tile.propTypes = {
+    /** Text label of the element. */
+    label: PropTypes.string,
+    /** The value of the element. */
+    value: PropTypes.number,
```
Comment (LeandroTorresSicilia) [inline packages/dashboard/src/components/Tile/index.js:80]: the value can be '1k'

File: packages/dashboard/src/components/Tile/index.js [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+    /** A CSS class for the outer element, in addition to the component's base classes. */
+    className: PropTypes.string,
+    /** An object with custom style applied for the outer element. */
+    style: PropTypes.object,
+};
+
+Tile.defaultProps = {
+    label: undefined,
+    value: undefined,
+    color: '#333333',
```
Comment (TahimiLeonBravo) [inline packages/dashboard/src/components/Tile/index.js:98]: it should work with the theme, we should use `getContastText`

File: packages/dashboard/docs/stories/Tile.story.js [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+    );
+};
+
+export const differentsColorsTite = () => {
+    const StyledContainer = styled.div`
+        padding: 10px;
+        background-color: #f4f6f9;
+        display: flex;
+        flex-wrap: wrap;
+    `;
```
Comment (LeandroTorresSicilia) [inline packages/dashboard/docs/stories/Tile.story.js:27]: you can move this outside

File: packages/dashboard/src/components/Tile/index.js [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+    backgroundColor: '#ffffff',
```
Comment (TahimiLeonBravo) [inline packages/dashboard/src/components/Tile/index.js:99]: it should work with the theme, we should use the `mainBackground`

**Review by TahimiLeonBravo — CHANGES_REQUESTED** [review]

**Pull request merged**